$NetBSD: patch-util_scan_Makefile,v 1.1 2020/04/20 00:41:43 joerg Exp $

perl doesn't look in the current directory anymore by default.

--- util/scan/Makefile.orig	2020-04-19 16:33:46.440330626 +0000
+++ util/scan/Makefile
@@ -25,7 +25,7 @@ $(binaries): $(objects)
 $(objects): atsc_psip_section.c atsc_psip_section.h
 
 atsc_psip_section.c atsc_psip_section.h:
-	perl section_generate.pl atsc_psip_section.pl
+	perl -I. section_generate.pl atsc_psip_section.pl
 
 include ../../Make.rules
 
